### PR TITLE
Install some basic dependencies using OS vendor

### PR DESCRIPTION
**Pull request by lagalbra**
@Microsoft/omsagent-devs 

Tested on an Oracle 7 machine (yum), Ubuntu 14 machine (apt-get), SLES 12 machine (zypper)
Unit, system, pBuild tests running now

### CODE REVIEW

**Comment by lagalbra** [comment]
Unit and system tests have passed.

**Comment by sugr4** [comment]
Sign off, assuming that install and upgrade scenarios were tested with this bundle on suse, debian based and rhel based machines.

File: installer/bundle/bundle_skel.sh [hunk 1]
```diff
@@ -370,6 +383,11 @@ EOF
     $tempFile 1> /dev/null 2> /dev/null
     [ $? -eq 0 ] && hasCtypes=0
     rm $tempFile
+
+    # Check #2: Attempt to run python with the single import command
+    python -c "import ctypes" 1> /dev/null 2> /dev/null
```
Comment (sugr4) [inline installer/bundle/bundle_skel.sh]: Maybe use > /dev/null 2>&1

Comment (lagalbra) [inline installer/bundle/bundle_skel.sh]: I noticed in the code that both forms are used; I may standardize this in a later review.

**Pull request merged**